cmd/queue/main_test.go

```diff
@@ -443,6 +443,38 @@ func TestQueueTraceSpans(t *testing.T) {
 	}
 }
 
+func BenchmarkProxyHandler(b *testing.B) {
+	var httpHandler http.HandlerFunc = func(w http.ResponseWriter, r *http.Request) {}
+	server := httptest.NewServer(httpHandler)
```

Comment:
We shouldn't actually need a proper downstream server here. That's just going to conflate the benchmark's metrics I think. Can we instead replace the proxies transport and not do any "proper" i/o? The activator's main handler is similarly tested AFAIK.